core/ui/src/views/DomainConfiguration.vue

```diff
@@ -1049,7 +1050,7 @@ export default {
       }
       this.currentProvider = provider;
 
-      if (this.domain.schema === "ad") {
+      if (this.domain.schema === "ad" && this.domain.location == "internal") {
```

Comment:
I don't understand this change, external providers shouldn't have delete button and shouldn't get to this point of the code :thinking:

Reply:
simple, we use the internal removal action for an external ldap provider